docs/guides/envvars.md

```diff
@@ -54,6 +54,27 @@ Note that `dotenv` is only for development, you should not use it in production,
 
 <docs-error>Do not commit your <code>.env</code> file to git, the point is that it contains secrets!</docs-error>
 
+If you're using the `@remix-run/cloudflare-pages` adapter, env variables work a little differently. When running locally, you'll need to manually pass your `.env` file to `wrangler` by doing this:
+
+```sh
+// package.json
+
+{
+	"scripts": {
+		"dev:wrangler": "cross-env NODE_ENV=development wrangler pages dev ./public --binding $(cat .env)"
+	}
+}
+```
+
+When deploying to Cloudflare, you can use the Cloudflare Pages Project Settings UI to set env variables for Preview and Production.
+
+Then, in your `loader` functions, both locally and when deployed on Cloudflare, you can access environment variables directly on `context`:
+```
+export const loader = async ({ context }) => {
+  console.log('My env:', context.ENV_VAR);
```

Comment:
Nit: Maybe it's just me, but I never like to leave a console log in the docs when the example could focus on more appropriate values. (I know my example in the thread included it, but docs examples tend to be taken more literally, folks will leave context logging in places it shouldn't be.) 

Maybe we can just focus on the env vars being passed verbatim by default, with an example like grabbing MY_SECRET_KEY out of `context.MY_SECRET_KEY`?

Reply:
Makes sense to me, however @ryanflorence's example above used the `console.log` so I figured I'd keep it consistent.